### PR TITLE
Update profile_side_switcher.css

### PR DESCRIPTION
**Pull request by lavigor**
Fix issue for mobile devices.

### CODE REVIEW

File: styles/prosilver/theme/profile_side_switcher.css [hunk 1]
```diff
@@ -20,7 +20,7 @@
 	background-image: none;
 }
 @media only screen and (max-width: 700px), only screen and (max-device-width: 700px) {
-	.leftsided div.postbody {
+	.section-viewtopic .leftsided div.postbody {
```
Comment (Galixte) [inline styles/prosilver/theme/profile_side_switcher.css]: hI @lavigor,

you have revert this fix: https://github.com/Tatiana5/profile_side_switcher/issues/9 could you fix with an other way please ?

**Comment by lavigor** [comment]
Hi, @Galixte.
If you see the fix carefully, I haven't touched the line from the fix #10.
Instead, #10 caused the issue with mobile style that have been reported to phpBB Guru support.

If there's another way to fix the issue, it shouldn't touch existing `prosilver`-based styles.

**Comment by Galixte** [comment]
It’s fixed for you ? Have you tested with Board3 portal extension and the welcome block for example ?

**Comment by lavigor** [comment]
I do not use such things, the same question for you.
If you provide me a link where these edits are installed and break something, I'll think about another solution.

But your statement about commit revert is wrong. No reverts at all.

**Comment by lavigor** [comment]
I also think that if the rules for `.section-viewtopic` are somehow used by the extensions that you've mentioned, this is the problem of those extensions.
Profile Side Switcher currently works wrong on the *clean installation* of phpBB without the fixes in this PR.

**Comment by Galixte** [comment]
ok i will check soon.

**Comment by Galixte** [comment]
It’s OK for me no problem with other pages or other extensions.

**Pull request merged**